### DISCUSSION TITLE
DripForm 对于自定义组件 如何通过formRef获取到值？

### DISCUSSION THREAD

**Discussion by cyue177** [jd-opensource/drip-form#195] [Q&A]
如题

**Comment by mengshang918** [comment]
1. 可以描述下自定义组件的场景以及需要使用哪些数据
2. 目前为了性能考虑，只透传了当前表单的数据。
    -   目前[roadmap](https://github.com/orgs/JDFED/projects/2/views/7)中已有规划，在保证性能的前提下去透传全局数据。
    -   如果目前需要使用全局数据，有个hack的方法，目前透传了`globalformdatastorekey`字段,可以通过`window[globalformdatastorekey]`获取到全局数据。
          **注意：** `globalformdatastourekey`只为临时方法，后续v0.9.0开发后会替代
